static/css/around-map.css

```diff
@@ -295,63 +295,34 @@ footer {
   background-color: #00BF6BEE;
 }
 
-/* ===================================== */
-/* OSM & Preact build CSS into html tag  */
-
-.mapboxgl-canvas-container {
-  position: fixed !important;
-}
-
-.super-card {
-  border: 0.2rem solid #3ba77c !important;
-}
-
-.first-card {
-  order: 0 !important;
-}
-
-.current-marker {
-  border: 0.4rem solid #3ba77c !important;
-  z-index: 1;
-}
-
 .visible {
-  display: flex !important;
+  display: flex!important;
 }
 
-/* ===================================== */
-
-.super-marker {
-  background-image: url('/static/images/marker-green.png');
-  background-size: cover;
-  cursor: pointer;
-  width: 2rem;
-  height: 3rem;
-  border-radius: 0;
-  background-color: transparent;
-  border: transparent;
-  z-index: 2;
-  left: -0.4rem;
-  top: -2rem;
+.first-card {
+  order: 0;
 }
 
 .current-card > h2 {
   color: #3ba77c;
 }
 
-.marker.current {
-  background-image: url('/static/images/marker-red.png');
-  z-index: 1;
+.super-card {
+  border: 0.2rem solid #3ba77c;
 }
 
-.mapboxgl-popup {
-  width: 50%;
-  z-index: 2;
-  top: -1.9rem;
+.marker {
+  fill: #297ddb;
 }
-
-.mapboxgl-popup-content p {
-  text-align: center;
+.current-marker {
+  fill: #9B9B9B;
+  /* need important to overide leaflet */
+  z-index: 9999!important;
```

Comment:
leaflet goes up to 1010 if I remember well. You may want to set it to that level instead.